astro/src/components/get-started/DelegateStep3.astro

```diff
@@ -0,0 +1,20 @@
+---
+import Code from 'astro/components/Code.astro';
+
+---
+<dd>
```

Comment:
I didn't understand why we used this tag here when we didn't for other steps.

Reply:
Fixed